monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -21,52 +25,68 @@ class FileUpload(flask_restful.Resource):
     File upload endpoint used to exchange files with filepond component on the front-end
     """
 
+    def __init__(self, file_storage_service: IFileStorageService):
+        self._file_storage_service = file_storage_service
+
+    # TODO: Fix references/coupling to filepond
+    # TODO: Add comment explaining why this is basically a duplicate of the endpoint in the
```

Comment:
It has the same code, but sending files to agent and to user are different use cases, worth considering not merging this into one resourse

Reply:
I'll leave a comment to that effect in a future PR.